libnmstate/nm/sriov.py

```diff
@@ -148,8 +135,7 @@ def _has_sriov_capability(ifname):
 
 def get_info(device):
     """
-    Provide the current active SR-IOV total-vfs runtime value and the live
-    configuration for each VF for a device.
+    Provide the current active SR-IOV runtime values
     """
     info = {}
```

Comment:
nit: I think this one is redundant now.
You can just create it on return (e.g. `return {Ethernet.SRIOV_SUBTREE: sriov_config}`)

Reply:
Done, thanks!